modules/yieldoneBidAdapter.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+```
+Module Name: YIELDONE Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: y1dev@platform-one.co.jp
+```
+
+# Description
+
+Connect to YIELDONE for bids.
+
+THE YIELDONE adapter requires setup and approval from the Rubicon Project team. Please reach out to your account team or y1s@platform-one.co.jp for more information.
+
+# Test Parameters
+```
+  var adUnits = [{
+    code: 'banner-ad-div',
+    sizes: [300, 250],
+    bids: [{
+    bidder: 'yieldone',
+      params: { 
+        placementId: '44082',
+        width:'300',
```

Comment:
why do you need repeat width/height params when you can get this off the bid.sizes?